src/pudl/package_data/settings/etl_full.yml

```diff
@@ -239,7 +239,105 @@ datasets:
           2021,
           2022,
         ]
-      eia860m: true
+      eia860m_year_months:
```

Comment:
this is a looooot of year_months in the settings here... i could convert to using "all" like CEMS.